INSTALL.md

```diff
@@ -73,18 +78,32 @@ reports that Flambda is available, some further optimization options
 can be used; see the entry about `-flambda-opts` in the build guide
 for more details.
 
-Build and Installation Procedure
---------------------------------
+Choice of Build and Installation Procedure
+------------------------------------------
+
+There are two partially overlapping infrastructures available to build
+Coq. They are available through `Makefile.make` (legacy / hybrid
+build) and `Makefile.dune` (full Dune build).
```

Comment:
Since `Makefile.make` is the legacy option, perhaps it should be mentioned after `Makefile.dune`?

Reply:
I mentioned it first because it's the default.